src/AddToBucket.jsx

```diff
@@ -3,6 +3,17 @@ import { createSignal } from "solid-js";
 export function AddToBucket(props) {
   const [newItem, setNewItem] = createSignal('');
 
+  //  Save new wish to localstorage
+  function saveToLocalStorage(prev, newWish){
+    const allWishes = [
+      ...prev, {
+        text: newWish,
+        complete:false,
+      }
+    ].reverse()
+    localStorage.setItem("solid-bucket-list", JSON.stringify(allWishes))
```

Comment:
@Eth3R-One Let's create a utility file for the lcalstorage related operations... u can create a util folder under src and create the utility under that.